frontend/src/scenes/paths/pathsLogic.ts

```diff
@@ -183,6 +185,42 @@ export const pathsLogic = kea<pathsLogicType<PathNode, PathResult>>({
                 filters: { ...values.filter, path_start_key, path_end_key, path_dropoff_key },
             })
         },
+        viewPathToFunnel: ({ pathItemCard }) => {
+            const events = []
+            let currentItemCard = pathItemCard
+            while (currentItemCard.targetLinks.length > 0) {
+                const name = currentItemCard.name.includes('http')
+                    ? '$pageview'
+                    : currentItemCard.name.replace(/(^[0-9]+_)/, '')
+                events.push({
+                    id: name,
+                    name: name,
+                    type: 'events',
+                    order: currentItemCard.depth - 1,
+                    ...(currentItemCard.name.includes('http') && {
+                        properties: [
```

Comment:
setting this property manually feels really hacky, are there any reuseable methods we currently use to do this in our app?